blosc2_btune/__init__.py

```diff
@@ -91,5 +91,18 @@ def set_params_defaults(**kwargs):
     lib.set_params_defaults(*args)
 
 
+class ReuseModels:
+    def __enter__(self):
+        libpath = get_libpath()
+        lib = ctypes.cdll.LoadLibrary(libpath)
+        lib.set_reuse_models(True)
+
+
+    def __exit__(self, exc_type, exc_value, exc_tb):
+        libpath = get_libpath()
+        lib = ctypes.cdll.LoadLibrary(libpath)
```

Comment:
Instead of duplicating the `ctypes.cdll.LoadLibrary` call, I suggest to add a `ReuseModels.__init__()` method and do the initialization there.

Reply:
Fixed in 418b718b681a19517df7a542b1c783ae8257440b